### PR TITLE
Replace remove files activity

### PR DESCRIPTION
**Pull request by djjuhasz**
Replace "github.com/artefactual-sdps/remove-files-activity" with "github.com/artefactual-sdps/temporal-activities/removefiles".

Also:
- Remove example ".remove" and ".succumb" files
- Remove creation of ".remove" file from kube dev overlay
- Add unit tests for the preprocessing workflow

### CODE REVIEW

**Comment by djjuhasz** [comment]
Waiting on https://github.com/artefactual-sdps/temporal-activities/pull/7 being merged to allow setting RemoveNames from a config value.

File: hack/kube/overlays/dev/preprocessing-moma-secret.yaml [hunk 1]
```diff
@@ -18,5 +18,6 @@ stringData:
 
     [worker]
     maxConcurrentSessions = 1
-  .remove: |
-    .DS_Store
+
+    [removefiles]
```
Comment (jraddaoui) [inline hack/kube/overlays/dev/preprocessing-moma-secret.yaml:22]: I'd use `removeFiles` for consistency.

File: internal/config/config_test.go [hunk 2]
```diff
@@ -11,8 +11,17 @@ import (
...
+verbosity = 2
+sharedPath = "/tmp"
 [temporal]
 address = "host:port"
+namespace = "default"
+taskQueue = "preprocessing"
+workflowName = "preprocessing"
+[worker]
+maxConcurrentSessions = 1
+[removefiles]
```
Comment (jraddaoui) [inline internal/config/config_test.go:23]: Same.

**Review by jraddaoui — APPROVED** [review]
Nice!

**Pull request merged**